tests/console/python_scientific.pm

```diff
@@ -33,6 +33,7 @@ sub run_python_script {
             record_soft_failure("$failmsg");
         }
     }
+    script_run("rm '$script'");
```

Comment:
I believe this should go into the `post_run_hooks` and the `post_fail_hooks`, otherwise the file remains present, also if the test run fails.

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/sshd.pm#L221 and https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/sshd.pm#L227 as an example.

Reply:
@grisu48 , Thanks for reviewing and pointing to the correct way of implement this.
Please check the updated commit.